src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx

```diff
@@ -23,7 +23,21 @@ interface IGroupComparisonMutationsTabPlotProps {
 }
 
 function plotYAxisLabelFormatter(symbol?: string, groupName?: string) {
-    return `${symbol} ${groupName} Mutations`;
+    // lowercase = 1 and uppercase = 1.3 (based on 'w' and 'W'), if groupName >= 13, stop and + "..."
```

Comment:
weird that the params of this function are optional    lets see what happens if we make them required

Reply:
@gblaih

Reply:
Making them required doesn't seem to change anything so I will change it to that.